### PR TITLE
Add a little more rares

### PR DESCRIPTION
**Pull request by libraryaddict**

### CODE REVIEW

File: mafia/data/raretiles.json [hunk 1]
```diff
@@ -805,6 +821,7 @@
   { "minute": 8093, "row": 2, "column": 6 },
   { "minute": 8093, "row": 6, "column": 4 },
   { "minute": 8111, "row": 5, "column": 7 },
+  { "minute": 8129, "row": 6, "column": 9 },
```
Comment (docrostov) [inline mafia/data/raretiles.json:824]: nope this one's wrong sorry

Comment (docrostov) [reply mafia/data/raretiles.json:824]: (this is an irrat-style joke where i just say something untrue to be a dick. sorry irrat. this is a good PR, you are doing good work, thank you for your service.)

**Review by docrostov — APPROVED** [review]
hell yeah this rules

File: mafia/data/raretiles.json (hunk 1 above)
Comment (libraryaddict) [reply mafia/data/raretiles.json:824]: That doesn't even make sense

**Pull request merged**